src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { PlaylistContext } from '../Context/PlaylistContext';
+import { spotifySearch } from '../../util/Spotify';
+import './SearchBar.css'
+
+export default function SearchBar(){
+
+    const infoContext = useContext(PlaylistContext);
+
+    function submitSearch(search) {
+        //sendSearch -> send to spotify API
+        spotifySearch(search).then(searchResults => {
+            infoContext.setSearchResult(searchResults);
+        });
+        console.log('search criteria: ' + search + ' - submited to spotify API')
+    }
+
+    function handleTermChange(event) {
+        infoContext.setSearch(event.target.value)
+    }
+
```

Comment:
The `search` variable is only used inside this component. Should be part of the Component's state instead of the Context